google/cloud/aiplatform/metadata/execution.py

```diff
@@ -31,6 +34,51 @@ class _Execution(_Resource):
     _resource_noun = "executions"
     _getter_method = "get_execution"
 
+    def add_artifact(
```

Comment:
shall we call this add_execution_events?

Reply:
The BE API method was called add_execution_event as it takes event as the input instead of the artifact. Here we basically hide the event concept and directly input artifact as the input so I think add_artifact is more proper. Also, as this is an instance method in the exeuction class, I think we do not need to have execution in the name either to avoid duplication.

Reply:
SG